results/cmd/api/main.go

```diff
@@ -81,3 +81,27 @@ func (s *server) CreateTaskRun(ctx context.Context, req *pb.CreateTaskRunRequest
 	}
 	return taskrunFromClient, nil
 }
+
+// UpdateTaskRun receives TaskRun and FieldMask from client and uses them to update records in local Sqlite Server.
+func (s *server) UpdateTaskRun(ctx context.Context, req *pb.UpdateTaskRunRequest) (*pb.TaskRun, error) {
+	database := s.db
```

Comment:
nit: It looks like this is only used once, so I'm not sure if this is strictly necessary (e.g. we could just use `s.db.Prepare` below instead). What do you think?

Reply:
done, I have deleted this line